src/main/java/com/commercetools/sync/commons/utils/SyncUtils.java

```diff
@@ -11,6 +11,8 @@
 import javax.annotation.Nullable;
 
 public final class SyncUtils {
+  private static final String DOUBLE_QUOTE = "\"";
+  private static final String DOUBLE_QUOTE_WITH_ESCAPE_CHARACTERS = "\\\\\"";
```

Comment:
Why do we need this and the one above?

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/726/commits/672a324a4c2e4044fd1c098d08238ec10860c5e3